daml/Marketplace/Clearing/Role.daml

```diff
@@ -79,11 +79,13 @@ template Offer
     signatory operator
 
     controller provider can
-      Accept : ContractId Role
+      Accept : (ContractId Role, ContractId Market.Service)
         with
           ccpAccount : Account
         do
-          createOrLookup Role with ..
+          marketCid      <- createOrLookup Market.Service with customer = provider, ..
+          roleCid        <- createOrLookup Role with ..
```

Comment:
generally when you're aligning <- or = in Daml it's best to do so from the variable with the longest characters, so in this case it would be:
```haskell
marketCid <- createOrLookup etc
roleCid   <- createOrLookup etc
```